src/pear/script.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+## VIASH START
+## VIASh END
+
+[[ "$par_emperical_freqs" == "false" ]] && unset par_emperical_freqs
+[[ "$par_nbase" == "false" ]] && unset par_nbase
+
+if [[ "${par_forward_fastq##*.}" == "gz" ]]; then
+  gunzip $par_forward_fastq
+  par_forward_fastq=${par_forward_fastq%.*}
+fi
+if [[ "${par_reverse_fastq##*.}" == "gz" ]]; then
+  gunzip $par_reverse_fastq
+  par_reverse_fastq=${par_reverse_fastq%.*}
+fi
+
+par_output="pear_output"
```

Comment:
```suggestion
par_output=$(mktemp -d -p "$meta_temp_dir" pear_XXXXXXXXX)
```

Also, I'd suggest renaming `par_output` to `output_dir`, because it looks like `par_output` is an argument of the `pear` component

Reply:
This was giving me an error. Apparently the template needs to use a `.` instead of a `_`.